src/main/java/org/example/persistence/ormanager/ORManagerImpl.java

```diff
@@ -111,9 +111,10 @@ public <T> Optional<T> findById(Serializable id, Class<T> cls) {
              PreparedStatement ps = connection.prepareStatement(getTableNameForSelect(cls))) {
             ps.setLong(1, (Long) id);
             ResultSet rs = ps.executeQuery();
+            ResultSetMetaData rsMetaData = rs.getMetaData();
             while (rs.next()) {
-                long personId = rs.getLong("id");
-                String firstName = rs.getString("first_name");
+                long personId = rs.getLong(rsMetaData.getColumnName(1));
+                String firstName = rs.getString(rsMetaData.getColumnName(2));
```

Comment:
Did the tests work? Just asking because we got a "serialVersionUID" field and don't know if the first rsMetaData.getColumnName() should be 1 or 2.

Reply:
It works, the array starts from 0 so 1 is the id field